README.md

```diff
@@ -79,29 +65,38 @@ Click the **Deploy Azure** button to  begin.
-5. Enter a Username for the **domainAdminUsername**. This is a new account.
-6. Enter a password for the **domainAdminPassword**. This is a new password.
-7. Enter a **domainName** and ensure it finishes in **.com**.
-8. Enter your **AzureAdminUsername**. This must be the same account you logged into from step 1.
-9. Enter your **AzureAdminPassword**. This must be the same password you used to log in from step 2.
-10. Enter the CAS license registration code for the **registrationCode**.
-11. Use the default addresses that are pre-entered for the **adminVMBlobSource** and **_artifactsLocation**. 
-12. Read the Terms and Conditions and once you are satisified with the information you have entered click the **I Agree** icon.
-13. Click **Purchase** to begin deployment.
+1. Launch an Azure Cloud Shell Powershell instance. See [Overiew of Azure Cloud Shell](https://docs.microsoft.com/en-ca/azure/cloud-shell/overview?view=azurermps-5.0.0) for details on how to set this up. The rest of the setup steps will be run in the Azure Cloud Shell. It is recomended to maximize the Cloud Shell window so prompts can be properly seen.
+1. Change to your Cloud Drive directory. 
+    ```
+    cd $HOME\CloudDrive\
+    ```
+1. Download the CAM Deployment script.
+    ```
+    Invoke-WebRequest -URI https://raw.githubusercontent.com/teradici/deploy/bd/Deploy-CAM.ps1 -OutFile .\Deploy-CAM.ps1
+    ```
+1. Run the CAM Deployment script.
+    ```
+    .\Deploy-CAM.ps1
+    ```
+1. You may now be prompted to select a subscription to use to deploy CAM in. Choose the number accordingly.
+1. Exisiting Resource Groups will be listed if there are Resource Groups available for CAM to be deployed into. Either select an existing Resource Group by selecting the number next to desired Resource Group or create a new Resource Group by providing a name for the new Resource Group.
+1. Available deployment locations will be listed. Select a location from one of the listed options by typing in the *Location* field for the desired location.
+1. Enter the desired credentials for the **Administrative Domain User**. This is a new account that will be created.
+1. Enter a **Domain Name** and ensure it finishes in *.com*.
+1. Enter the CAS license registration code for the **Registration Code**.
+1. When prompted for Service Principal credentials, either press 'Enter' to continue and let the script generate them or enter 'no' to provide manually generated credentials.
 
 The deployment will now begin to run. 
 
 You can track it through the notifications icon or for a more detailed view of your deployment click the **Resource Groups** icon in the Azure portal and click on your resource group.
 
+**NOTE:** The Azure Cloud Shell will disconnect after 20 minutes of inactivity. This is expected behaviour. Once the deployment has begun, Azure will handle processing the deployment and the Azure Cloud Shell is no longer needed.
 
 ## Known Issues with Deploying the Solution
```

Comment:
I think we have some better text for all this in the admin guide - like, we don't spell out the regions anymore and refer to the Azure availability page.